### PR TITLE
Include code around PUE notice for Invalid Licenses

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#129726](http://central.tri.be/issues/129726)
---

### CODE REVIEW

File: src/Tribe/Admin/Notice/Plugin_Download.php [hunk 1]
```diff
@@ -96,10 +106,14 @@ public function show_inactive_plugins_alert() {
 			'a'      => array( 'href' => array() ),
 		);
 
+		$pue_notice_html = '<p>' . esc_html__( '---MESSAGE_PLACEHOLDER---', 'tribe-common' ) . '</p>';
```
Comment (sc0ttkclark) [inline src/Tribe/Admin/Notice/Plugin_Download.php]: What use is the translation aspect here?

Comment (bordoni) [reply src/Tribe/Admin/Notice/Plugin_Download.php]: I am waiting for Leah's message here. That is why I left a place holder there. =)

Comment (sc0ttkclark) [reply src/Tribe/Admin/Notice/Plugin_Download.php]: Gotcha, thanks! I put it on hold but code looks good to me!

File: src/Tribe/Admin/Notice/Plugin_Download.php [hunk 2]
```diff
@@ -96,13 +106,23 @@ public function show_inactive_plugins_alert() {
 			'a'      => array( 'href' => array() ),
 		);
 
+		$plugin_names_clean_text = wp_kses( $this->implode_with_grammar( $plugin_name ), $allowed_html );
+		$req_plugin_names_clean_text = wp_kses( $this->implode_with_grammar( $req_plugins ), $allowed_html );
+
+		$notice_html_content = '<p>' . esc_html__( 'To begin using %2$s, please install and activate the latest version of %3$s.', 'tribe-common' ) . '</p>';
+
+		$read_more_link = '<a href="http://m.tri.be/1aev" target="_blank">' . __( 'Read more.', 'tribe-common' ) . '</a>';
```
Comment (sc0ttkclark) [inline src/Tribe/Admin/Notice/Plugin_Download.php]:
```suggestion
		$read_more_link = '<a href="http://m.tri.be/1aev" target="_blank">' . esc_html__( 'Read more.', 'tribe-common' ) . '</a>';
```

File: src/Tribe/Admin/Notice/Plugin_Download.php [hunk 3]
```diff
@@ -96,13 +106,23 @@ public function show_inactive_plugins_alert() {
...
+		$pue_notice_text = __( 'There’s a new version of %1$s available, but your license is expired. You’ll need to renew your license to get access to the latest version. If you plan to continue using your current version of the plugin(s), be sure to use a compatible version of The Events Calendar. %2$s', 'tribe-common' );
```
Comment (sc0ttkclark) [inline src/Tribe/Admin/Notice/Plugin_Download.php]:
```suggestion
		$pue_notice_text = esc_html__( 'There’s a new version of %1$s available, but your license is expired. You’ll need to renew your license to get access to the latest version. If you plan to continue using your current version of the plugin(s), be sure to use a compatible version of The Events Calendar. %2$s', 'tribe-common' );
```

**Comment by bordoni** [comment]
Ready for another set of eyes.

**Pull request merged**